bundle/language_it.properties

```diff
@@ -1175,7 +1175,9 @@ angal.patient.note
 angal.patient.parenttogether                                                                           = I genitori sono insieme
 angal.patient.patientbrowser.title                                                                     = Elenco Pazienti
 angal.patient.patientphoto                                                                             = Foto
-angal.patient.patientselection.title                                                                   = Selezione Paziente
+angal.patient.patientphoto.error.title                                                                 = Patient Photo Error
+angal.patient.patientphoto.error.filenotfound                                                          = Errore della foto del paziente
+angal.patient.patientselection.title                                                                   = Non � stata trovata la foto del paziente.
```

Comment:
There's no need to provide translations as they will come from Transifex once merged on develop, thanks